contrib/hawq-docker/centos7-docker/hawq-test/conf/capacity-scheduler.xml

```diff
@@ -0,0 +1,134 @@
+<!--
```

Comment:
What's the difference between these configuration xml files under centos7-docker and centos6-docker ?  If they are the same, please use one copy and point out the right path when COPY them to the docker image. DRY. If they have little differences, we could use most of the common files and just store the different copies. Otherwise you could use 2 copies for every conf files.